build.cake

```diff
@@ -29,34 +26,29 @@ Task("Restore-NuGet-Packages")
     .IsDependentOn("Clean")
     .Does(() =>
 {
-    NuGetRestore("./src/Example.sln");
+    DotNetCoreRestore("./src/Example.sln");
 });
 
 Task("Build")
     .IsDependentOn("Restore-NuGet-Packages")
     .Does(() =>
 {
-    if(IsRunningOnWindows())
-    {
-      // Use MSBuild
-      MSBuild("./src/Example.sln", settings =>
-        settings.SetConfiguration(configuration));
-    }
-    else
+    DotNetCoreBuild("./src/Example.sln", new DotNetCoreBuildSettings
     {
-      // Use XBuild
-      XBuild("./src/Example.sln", settings =>
-        settings.SetConfiguration(configuration));
-    }
+      Configuration = configuration,
+      NoRestore = true,
+    });
 });
 
 Task("Run-Unit-Tests")
     .IsDependentOn("Build")
     .Does(() =>
 {
-    NUnit3("./src/**/bin/" + configuration + "/*.Tests.dll", new NUnit3Settings {
-        NoResults = true
-        });
+    DotNetCoreTest("./src/Example.sln", new DotNetCoreTestSettings
+    {
+      Configuration = configuration,
+      NoRestore = true,
```

Comment:
Looks like indentation is off (`2` spaces instead of `4`?)

Reply:
I've added a shared `.editorconfig` and reformatted.